app/src/components/button/Button.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {Props} from './types';
+import './Button.scss';
+
+function Button(props: Props): JSX.Element {
```

Comment:
Предлагаю

```suggestion
function Button({children, className = 'btn t-main'}: Props): JSX.Element {
```

И убрать `Button.defaultProps` ниже